### PR TITLE
Update Intel tools to oneAPI 2025.0

### PR DESCRIPTION
**Pull request by fwyzard**

### CODE REVIEW

File: intel-vtune.spec
Comment (fwyzard) [inline intel-vtune.spec:1]: I'm not sure if this file is used or not.
If it's not used, I guess we should remove it ?

**Comment by fwyzard** [comment]
please test

File: intel-vtune.spec
Comment (smuzaffar) [reply intel-vtune.spec:1]: @fwyzard , currently it is used only to instruct the system to include the `scram-tools.file/tools/intel-vtune/intel-vtune.xml` file. We can move `scram-tools.file/tools/intel-vtune/intel-vtune.xml` under `scram-tools.file/tools/systemtools/intel-vtune.xml` and drop `intel-vtune.spec`. `systemtools` contains the collections of all tool files which we take from system

Comment (fwyzard) [reply intel-vtune.spec:1]: OK, I see.
I would prefer to keep the spec file to define the variables, remove the generation of the xml from the spec, and change the xml file to use `@TOOL_VERSION@` and `@TOOL_ROOT@`.

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
@smuzaffar I don't see any changes in the area I get from `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmsdist/9503/42639/install.sh`.
Is it because these tools are not enabled by default ?

**Comment by smuzaffar** [comment]
> Is it because these tools are not enabled by default ?

correct @fwyzard , tool is not enabled by default . It is available in the release  /cvmfs/cms-ci.cern.ch/week0/cms-sw/cmsdist/9503/42639/CMSSW_14_2_X_2024-11-06-2300/config/toolbox/el8_amd64_gcc12/tools/available/intel-vtune.xml but one needs to do `scram setup intel-vtune` to activate it.

**Comment by fwyzard** [comment]
> It is available in the release /cvmfs/cms-ci.cern.ch/week0/cms-sw/cmsdist/9503/42639/CMSSW_14_2_X_2024-11-06-2300/config/toolbox/el8_amd64_gcc12/tools/available/intel-vtune.xml

Mhm, but that is still the old version (2018.0.2.525261), not the version that was supposed to come from this PR (2025.0).

**Comment by smuzaffar** [comment]
ah right, for PR tests we only copy [tools/selected](https://github.com/cms-sw/cms-bot/blob/master/pr_testing/test_multiple_prs.sh#L650) directory from the newly build toolconf in to cmssw dev area. The new intel-vtune.xml is in the actual cmssw-tool-conf directory 
```
> cat /cvmfs/cms-ci.cern.ch/week0/PR_0beb84cf/el8_amd64_gcc12/cms/cmssw-tool-conf/100.0-b7bf6ee62264649ece22c02126455f83/tools/available/intel-vtune.xml
<tool name="intel-vtune" version="2025.0">
  <info url="https://www.intel.com/content/www/us/en/developer/tools/oneapi/vtune-profiler.html"/>
  <client>
    <environment name="INTEL_VTUNE_BASE" default="/cvmfs/projects.cern.ch/intelsw/oneAPI/linux/x86_64/2025/vtune/2025.0"/>
    <environment name="BINDIR" default="$INTEL_VTUNE_BASE/bin64"/>
  </client>
  <runtime name="PATH" value="$INTEL_VTUNE_BASE/bin64" type="path"/>
  <runtime name="VTUNE_PROFILER_DIR" value="$INTEL_VTUNE_BASE"/>
  <runtime name="VTUNE_PROFILER_2025_DIR" value="$INTEL_VTUNE_BASE"/>
</tool>
```

I can update bot code to also copy extra available tools

**Comment by fwyzard** [comment]
Thanks, the new tool looks good to me, so we can merge the PR.

> I can update bot code to also copy extra available tools

Thanks - it's not urgent, but could be handy.

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**